### DISCUSSION TITLE
Use Superbooga

### DISCUSSION THREAD

**Discussion by mykeehu** [oobabooga/text-generation-webui#2300] [General]
I am interested in using superbooga. I've read about backward logic, but I don't understand it. Can someone give me a concrete example of how I can use superbooga during chat? 
Can I type in properties or remind my character when he gets lost "sorry, but you have a blue jacket"? Or do I just type "blue jacket" and send it? How do I see the features, weights? Can I load the properties of the characters?

**Comment by somenewaccountthen** [comment]
What i do sometimes is thing like: What a nice blue jacket *  character thinks the blue jacket is nice*
use name of character

**Reply by mykeehu** [reply]
Can I use it so that if I get an incorrect answer (for example, it says she's supposed to be wearing a skirt, but she's wearing pants), I can type "(char)'s wearing a skirt" in superbooga, send it, and then regenerate the answer? Or is it even better to type that before sending my own comment?

**Reply by somenewaccountthen** [reply]
YOU CAN!! YOU CAN!!.. try :)
I am not sure what you mean but i think yes. If it fails regenerate with the "* <force something to be fact e.g:  Bobby is wearing a skirt> *"  added

(That's how i see it. All 2ndlanguage, i am sure there's a fancy name that describes just this)
